metrics/src/main/scala/com/mesosphere/usi/metrics/dummy/DummyMetrics.scala

```diff
@@ -0,0 +1,48 @@
+package com.mesosphere.usi.metrics.dummy
```

Comment:
Where is that used? I'm not sure I understand the relation to this pull request.

Reply:
Needed for testing of e.g. `ZookeeperPersistenceStore` which uses some metrics.